src/version.ts

```diff
@@ -0,0 +1,24 @@
+import get from 'axios';
+
+const API_URL = 'https://api.github.com/repos/TaneAkashi/DakokuApp/tags';
+
+type Tag = {
+  name: string;
+  zipball_url: string;
+  tarball_url: string;
+  commit: {
+    sha: string;
+    url: string;
+  };
+  node_id: string;
+};
+
+const getVersions = async (): Promise<string[]> => {
+  const tags: Tag[] = (await get(API_URL)).data;
+  return tags.map((tag) => tag.name.slice(1));
+};
+
+export const getLatest = async (): Promise<string | null> => {
+  const versions: string[] = await getVersions();
```

Comment:
```suggestion
  const versions = await getVersions();
```

型推論で不要かなーと
getVersions で返す型を指定しているので